### PR TITLE
Hide download CSV link when in the tour

### PR DESCRIPTION
**Pull request by quis**
There is really no point to downloading the CSV file of delivery info when you’re in the tour. It’s just distracting at this point.

### CODE REVIEW

File: app/templates/partials/jobs/notifications.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <div class="dashboard-table">
 {% endif %}
 
-  {% if notifications %}
+  {% if notifications and request.args.get('help', '0') == '0' %}
```
Comment (ashimali) [inline app/templates/partials/jobs/notifications.html]: Why not pass this to partial as arg e.g. request.args.get('help', '0') == '0' becomes is_tour_page or true or false.

Comment (quis) [inline app/templates/partials/jobs/notifications.html]: Jinja macros can’t return a value unfortunately.

Comment (ashimali) [inline app/templates/partials/jobs/notifications.html]: Not quite what I meant. Just as notifications is set in the view function and passed to template you could have a variable "is_tour_page" set to result of request.args.get('help', '0') == '0' and pass that to template.  request.args.get('help', '0') == '0'  is a little opaque in this context.

Comment (quis) [inline app/templates/partials/jobs/notifications.html]: I follow you now. Changed in 29d9333.

**Comment by yahoopete** [comment]
:+1:

**Pull request merged**